csharp/test/Microsoft.ML.OnnxRuntime.Tests.Common/InferenceTest.cs

```diff
@@ -2039,6 +2039,8 @@ public SkipNonPackageTests()
             }
         }
 
+        // Test hangs on mobile.
```

Comment:
Don't know why it happens. It may be addressed by the change to return Task instead of void (guess) but I haven't had time to look into it.

AFAIK nobody uses the async API on mobile so low priority.